recipes/duckdb/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "duckdb" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cwida/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f3ee08f5a9422e5122e95624cbde0c1ce33416533f3eba7ba9be0cb320d613bd
+  patches:
+    - 0001-Support-building-with-std-c-14.patch
+    - 0002-Only-include-pytest-runner-in-setup_requires-if-we-r.patch
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: "cd tools/pythonpkg && $PREFIX/bin/python setup.py install"
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy 1.16
+    - pandas
+  run:
+    - python
+    - numpy
```

If you're using the numpy C API then you need this:
```suggestion
    - {{ pin_compatible('numpy') }}
```
See: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy